pkg/pillar/cmd/zedagent/handleProfile.go

```diff
@@ -161,15 +161,15 @@ func getLocalProfileConfig(getconfigCtx *getconfigContext, localServerURL string
 func writeOrTouchReceivedLocalProfile(getconfigCtx *getconfigContext, localProfile *profile.LocalProfile) {
 	if getconfigCtx.localProfile == localProfile.GetLocalProfile() &&
 		getconfigCtx.profileServerToken == localProfile.GetServerToken() {
-		touchProtoMessage(savedLocalProfileFile)
+		touchSavedConfig(savedLocalProfileFile)
 		return
 	}
 	contents, err := proto.Marshal(localProfile)
 	if err != nil {
 		log.Errorf("writeOrTouchReceivedLocalProfile Marshalling failed: %s", err)
 		return
 	}
-	writeProtoMessage(savedLocalProfileFile, contents)
+	writeConfig(savedLocalProfileFile, contents)
```

Comment:
Should this function be called writeSaveConfig since the other functions are *SavedConfig?

Reply:
I would just call it `saveConfig`... done